content/documentation/pages/6-recipes/9-cloud-providers/1-gke-regional-clusters.md

```diff
@@ -0,0 +1,407 @@
+
+Make the following modifications to the `server/server-deployment.yaml` file bumping up the replica count and adding the pod affinity:
+
+```
+spec:
+  replicas: 3
+```
+
+```
+  affinity:
+    podAffinity:
+      requiredDuringSchedulingIgnoredDuringExecution:
+      - labelSelector:
+          matchExpressions:
+          - key: app
+            operator: In
+            values:
+            - skipper
+        topologyKey: kubernetes.io/hostname
+```
+
+Deploy the manifests:
+
+```
+$ kubectl create -f server/server-config.yaml
+$ kubectl create -f server/server-svc.yaml
+$ kubectl create -f server/server-deployment.yaml
+```
+
+Get the nodes the pods are deployed to:
+
+```
+$ kubectl get pods -l app=scdf-server -o=custom-columns=NAME:.metadata.name,STATUS:.status.phase,NODE:.spec.nodeName
+NAME                           STATUS    NODE
+scdf-server-5ddf7bbd4f-dpnmm   Running   gke-regional-demo-default-pool-fb3e6608-0lx2
+scdf-server-5ddf7bbd4f-hlf9h   Running   gke-regional-demo-default-pool-e121c001-zhrt
+scdf-server-5ddf7bbd4f-vnjh6   Running   gke-regional-demo-default-pool-ea10f422-5f72
+```
+
+Verify the pods are deployed to the same nodes as Skipper:
+
+```
+$ kubectl get pods -l app=skipper -o=custom-columns=NAME:.metadata.name,STATUS:.status.phase,NODE:.spec.nodeName
+NAME                       STATUS    NODE
+skipper-6fd7bb796c-flm44   Running   gke-regional-demo-default-pool-e121c001-zhrt
+skipper-6fd7bb796c-l99dj   Running   gke-regional-demo-default-pool-ea10f422-5f72
+skipper-6fd7bb796c-vrf9m   Running   gke-regional-demo-default-pool-fb3e6608-0lx2
+```
+
+Verify connectivity to Data Flow:
+
+```
+$ SCDF_IP=$(kubectl get svc/scdf-server -o jsonpath='{.status.loadBalancer.ingress[*].ip}')
+$ curl -s http://$SCDF_IP/about | jq
+{
+  "featureInfo": {
+    "analyticsEnabled": true,
+    "streamsEnabled": true,
+    "tasksEnabled": true,
+    "schedulesEnabled": true,
+    "grafanaEnabled": true
+  },
+  "versionInfo": {
+    "implementation": {
+      "name": "spring-cloud-dataflow-server",
+      "version": "2.6.0.BUILD-SNAPSHOT"
+    },
+...
+...
+...
+```
+
+## Deploying Streams and Tasks
+
+Streams and Tasks deployed through Data Flow can also benefit from various placement options just as with server components. There may be situations where certain streams or tasks should only be deployed to specific zones or even nodes themselves. Deployed streams and tasks are applications just like any others so they also benefit from Kubernetes rescheduling them onto other nodes in the event of failures.
+
+Since streams and tasks are deployed to Kubernetes via Data Flow and Skipper, rather than applying these features to yaml manifests they are set through deployer properties. See the [Deployer Property](https://docs.spring.io/spring-cloud-dataflow/docs/current/reference/htmlsingle/#configuration-kubernetes-deployer) section of the Spring Cloud Data Flow reference manual for a full list of available deployer properties.
```

Comment:
```suggestion
Since streams and tasks are deployed to Kubernetes via Data Flow and Skipper, rather than applying these features to YAML manifests, they are set through deployer properties. See the [Deployer Property](https://docs.spring.io/spring-cloud-dataflow/docs/current/reference/htmlsingle/#configuration-kubernetes-deployer) section of the Spring Cloud Data Flow reference manual for a full list of available deployer properties.
```